docs/docs/integrations/plus.md

```diff
@@ -52,3 +50,25 @@ Snapshots must be enabled to be able to submit examples to Frigate+
 You can view all of your submitted images at [https://plus.frigate.video](https://plus.frigate.video). Annotations can be added by clicking an image.
 
 ![Annotate](/img/annotate.png)
+
+## Use Models
+
+Models available in Frigate+ can be used with a special model path. No other information needs to be configured for Frigate+ models because it fetches the remaining config from Frigate+ automatically.
+
+```yaml
+model:
+  path: plus://e63b7345cc83a84ed79dedfc99c16616
+```
+
+Models are downloaded into the `/config/model_cache` folder and only downloaded if needed.
+
+You can override the labelmap for Frigate+ models like this:
+
+```yaml
+model:
+  path: plus://e63b7345cc83a84ed79dedfc99c16616
+  labelmap:
+    3: animal
+    4: animal
+    5: animal
```

Comment:
is the labelmap going to be documented somewhere so it's clear which index corresponds with which label?

Reply:
I plan to make it visible in Frigate+. It's also in the JSON file inside model_cache.